rfd/0113-moderated-file-transfers.md

```diff
@@ -0,0 +1,176 @@
+
+---
+authors: Michael Myers (michael.myers@goteleport.com)
+state: draft
+---
+
+# RFD 113 - Web Moderated Session File Transfer
+
+## Required approvers
+
+Engineering: @zmb3 @xacrimon @jakule
```

Comment:
Does the security and/or product team need to approve this as well?

Reply:
Yeah, wasn't sure who to add. Thanks for the nudge.